src/Test/Utilities/Portable/Compilation/IRuntimeEnvironment.cs

```diff
@@ -202,11 +231,22 @@ EmitOptions emitOptions
             EmitOptions emitOptions
         )
         {
+            if (emitOptions is null)
+            {
+                emitOptions = EmitOptions.Default.WithDebugInformationFormat(DebugInformationFormat.Embedded);
+            }
```

Comment:
Worry this isn't the right layer to be specifying emit options. It's pretty low and feels like more of a "just follow orders" layer of the stack. Did you consider putting this in a higher layer like `CompilationVerifier'? That seems to have the same single choke point.